topics/imaging/tutorials/yolo_prediction/tutorial.md

```diff
@@ -0,0 +1,287 @@
+> >    - *"IoU"*: Set to 0.45. This is used for Non-Maximum Suppression (NMS), which removes overlapping detections. A higher IoU value (e.g., 0.7) keeps more overlapping boxes. A lower IoU (e.g., 0.3) removes more overlaps, which may help clean up crowded images.
+> >    - *"Max detections"*: Set a reasonable cap like 300. This limits the number of objects detected per image.
+> > 
+> {: .comment}
+> 
+{: .hands_on}
+
+## 🧾 Explore the Outputs
+
+After running the tool, Galaxy will give you several output files for each image. Let’s go through what each one means and how to use them:
+
+📄 **Text files (*.txt):**
+
+These are plain text files containing the detection results. Each line in a file shows:
+
+```
+<class_id> <confidence_score> <x_center> <y_center> <width> <height>
+```
+
+For example: ```0 0.82 350 200 100 120```
+which means:
+
+ - Class ID 0 (in our case, Buccinidae)
+
+ - Detected with 82% confidence
+
+ - The bounding box is centered at (350, 200) and has a width of 100 and height of 120 (in pixels, relative to the image. You can use this file to do further analysis, like counting species or tracking locations over time.
+
+🖼️ **Overlay images (*.jpg):**
+
+These are your original images with colored boxes drawn around detected species. Each box also includes:
+The class name and the confidence score. For example, you might see a box labeled:
+`Crabe bythograeid 0.54`
+
+![buccinid](../../images/yolo/buccinid.jpeg)
+![bythog](../../images/yolo/bythog2.jpeg)
+
+
+
+These images are useful for visually checking whether detections are correct or if something was missed.
+
+⚠️ **No masks or segmentation files:** Since we used detect mode, this tool will not generate segmentation masks (like .tiff or polygon files). Those are only available in segment mode, which we'll cover next.
+
+
+
+## 💬 What to Look For
+
+- Are species detected correctly?
+- Any false positives or missed detections?
+- What confidence levels do you observe?
+- How many objects per image?
+
+---
+
+# 🧩 Part 2: Segmentation with an Ultralytics example
+
+Since the marine models only support detection, we’ll now demonstrate segmentation using a pretrained YOLOv8 model from Ultralytics.
+
+### 📦 Model & Input Image
+
+The YOLOv8n-seg model is a lightweight instance segmentation model trained on the COCO dataset (Common Objects in Context), which contains 80 everyday object classes such as person, bus, bicycle, car, dog, and more. These categories cover common scenes, making the model suitable for general-purpose detection and segmentation tasks. It’s ideal for quick testing, tutorials, or deployment on resource-limited systems.
+
+The class names file for this segmentation model is like:
```

Comment:
In line 157, you write that
> Each class name must be on its own line, in the same order used during model training.

As far as I understand, this means that the class names must be _identical_ to those used during training? If this is the case, the instructions should be phrased more strictly here:

```suggestion
The proper class names file for this segmentation model is:
```
But, if the class names must be identical, how should the reader cope with the ambiguity of `...` in line 230?